pkg/agent/openflow/pipeline.go

```diff
@@ -1853,6 +1853,25 @@ func (c *client) localProbeFlow(localGatewayIPs []net.IP, category cookie.Catego
 	return flows
 }
 
+// snatSkipNodesFlows installs the default flows for skip SNAT for traffic to
+// the k8s nodes.
+func (c *client) snatSkipNodesFlows(nodeIP net.IP, category cookie.Category) binding.Flow {
```

Comment:
If we just add a single flow, remove "s" from snatSkipNodesFlows

Reply:
Agreed